website/src/pages/roadmap.md

```diff
@@ -62,11 +62,11 @@ Next major release : [0.11.0](https://issues.apache.org/jira/projects/HUDI/versi
 |------------|--------|-----------|
 |JDBC Incremental Source GA   |0.11|[HUDI-1859](https://issues.apache.org/jira/browse/HUDI-1859)|
```

Comment:
@codope can you confirm if this is moved to 0.12 or not please? just saw https://issues.apache.org/jira/browse/HUDI-1862 was marked for 0.11

Reply:
Discussed with @codope over slack. We can make this change in a separate PR if needed.

Reply:
@xushiyan I'd prefer 0.12. Currently, metadata and multi-modal index are top priority. We can always add to 0.11 once we have some cycles to work on it.